host-configs/TOTAL/pangea3-gcc8.3-smpi-10.3.0.1.cmake

```diff
@@ -0,0 +1,121 @@
+# hostconfig for pangea3
+# module load cmake/3.18.2 gcc/8.3.0 cuda/11.3.0 smpi/10.3.0.1 openblas/0.3.8 lapack/3.9.0
+set(CONFIG_NAME "pangea3-gcc8.3-smpi-10.3.0.1" CACHE PATH "") 
+
+# Set up the tpls
+if (DEFINED ENV{GEOSX_TPL_DIR})
+  set(GEOSX_TPL_DIR ENV{GEOSX_TPL_DIR})
+else ()
+  message(FATAL_ERROR "You must set GEOSX_TPL_DIR")
+endif ()
```

Comment:
While you can surely work this way, I would encourage you to provide it with the command line.
```
python scripts/config-build.py -hc host-configs/your-platform.cmake -bt Release -D GEOSX_TPL_DIR=/full/path/to/thirdPartyLibs
```
see https://geosx-geosx.readthedocs-hosted.com/en/latest/docs/sphinx/QuickStart.html?highlight=GEOSX_TPL_DIR#compilation

Environment variables is surely acceptable for compilers, lapack... but for the `TPL` fixing it command line can be good (specially when you are playing with different versions and different `TPL` versions).

Reply:
Ok il remove the ENV part of this check.

Reply:
This does not work as the argument passed to cmake is not seen within the config file. I'll go back to ENV solution.

Reply:
Are you testing against
```
if (NOT DEFINED GEOSX_TPL_DIR})
```
Did you remove the trailing `}`?

I'm surprised that this is not working.
What is the content of the `cmake_cmd` which gets created by the `config-build.py` script?

Reply:
Yes I first though it was the forgotten } but I had the same without it.

Reply:
Hmm I didn't checkt the command, I'll retry.

Reply:
@XL64 What's the status of this PR?